src/Behat/Behat/Definition/Attribute/Definition.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Definition\Attribute;
+
+/**
+ * Marker interface for all Attributes regarding
+ * Call definitions
+ */
+interface Definition
```

Comment:
This interface should be marked `@internal` as the implementation does not actually support having more implementations

Reply:
It's been marked as internal.